### PR TITLE
Skipping MPI output for serial case in Pull Request Linux Driver.

### PR DESCRIPTION
**Pull request by jwillenbring**
@trilinos/framework 

## Description

We are adding a serial dev->master build. The PR driver was outputting some MPI variable info, and this caused the serial case to crash. I ifdef'ed the output to exclude the serial case.

### CODE REVIEW

File: cmake/std/PullRequestLinuxDriver.sh [hunk 1]
```diff
@@ -217,7 +217,10 @@ cmake --version
 
 module list
 
-echo "MPI type = sems-${SEMS_MPI_NAME:?}/${SEMS_MPI_VERSION:?}"
+#This crashes for the serial case since the MPI variables are not set
+if [ "Trilinos_pullrequest_gcc_4.9.3_SERIAL" != "${JOB_BASE_NAME:?}" ] ; then
```
Comment (mhoemmen) [inline cmake/std/PullRequestLinuxDriver.sh]: Why not just search for `_SERIAL` in the job name?

**Comment by william76** [comment]
@jwillenbring 	I saw this error too and fixed it on my PR #3624 at the same time you added this in, lol.  I'll remove the Automerge flag from this PR since I'm guessing they'll conflict since we basically changed the same line...

File: cmake/std/PullRequestLinuxDriver.sh (hunk 1 above)
Comment (william76) [reply cmake/std/PullRequestLinuxDriver.sh]: @mhoemmen I'm guessing is this is the easiest to do... If we add a 2nd SERIAL build in later then we'll probably fix that.

I did exactly the same thing Jim did on this too over on my PR branch, lol.  I think we must have seen the error at the same time :p

**Comment by jwillenbring** [comment]
Will’s PR #3624 has been merged, so this one can be closed.

**Pull request closed without merging**